Sources/WalletConnect/WalletConnectClient.swift

```diff
@@ -163,6 +162,18 @@ public final class WalletConnectClient {
         try sessionEngine.upgrade(topic: topic, permissions: permissions)
     }
     
+    /// For controller to extend a sequence lifetime
+    /// - Parameters:
+    ///   - topic: Topic of the sequence, it can be a pairing or a session topic.
+    ///   - ttl: Time in seconds that a target sequence is expected to be extended for. Must be greater than current time to expiry and for session lower than 7 days and for pairing lower than 30 days.
+    public func extend(topic: String, ttl: Int) throws {
```

Comment:
In the future we could add a constrained time enum as a parameter instead of the absolute time in seconds to make it simpler to call, like `extendBy: .month`